src/platform-aws.c

```diff
@@ -456,6 +459,13 @@ int platform_init(const char **provider_filter)
 		goto exit;
 	}
 
+	NCCL_OFI_INFO(NCCL_INIT | NCCL_NET, "Setting NCCL_NVLS_CHUNKSIZE to 512KiB");
+	ret = setenv("NCCL_NVLS_CHUNKSIZE", "524288", 0);
```

Comment:
In the previous cases, we first check if the user did explicitly set an env variable before we set them.
We are not doing this for `NCCL_NVLSTREE_MAX_CHUNKSIZE` and `NCCL_NVLS_CHUNKSIZE`, meaning that whatever the user sets gets ignored. Is this on purpose?

Reply:
I'd explained this in https://github.com/aws/aws-ofi-nccl/pull/332#discussion_r1491269250

Haven't gotten around to unifying usage yet, will do that after the release goes out.